www/gatsby-config.js

```diff
@@ -55,15 +55,17 @@ if (process.env.AIRTABLE_API_KEY) {
 
 if (process.env.ENABLE_LOCALIZATIONS) {
   dynamicPlugins.push(
-    ...langs.map(({ code }) => ({
-      resolve: `gatsby-source-git`,
-      options: {
-        name: `docs-${code}`,
-        remote: `https://github.com/gatsbyjs/gatsby-${code}.git`,
-        branch: `master`,
-        patterns: `docs/tutorial/**`,
-      },
-    }))
+    ...langs.map(({ code }) => {
```

Comment:
...is this necessary?

Reply:
This was came from eslint as a linting error. I just changed it to fix it.